equinix-openapi-metal/src/main/java/com/equinix/workflow/MetalGatewayOperator.java

```diff
@@ -63,7 +63,7 @@ public MetalGateway createMetalGatewayWithPrivateIpBlock(UUID projectId, String
      */
     public MetalGateway createMetalGateway(UUID projectId, CreateMetalGatewayRequest metalGatewayRequest)
             throws ApiException {
-        return metalGatewaysApi.createMetalGateway(projectId, metalGatewayRequest, RESPONSE_PAGE, RESPONSE_ITEMS_PER_PAGE).getMetalGateway();
+        return metalGatewaysApi.createMetalGateway(projectId, metalGatewayRequest, null, null, RESPONSE_PAGE, RESPONSE_ITEMS_PER_PAGE).getMetalGateway();
```

Comment:
The GitHub Actions workflow is acting like this change isn't in the PR; need to dig into what's going on. (`make docker_run` works locally for me)

Reply:
Updated the wrong copy of the file; during the build this file is copied over from `internal/workflow`.